README.md

```diff
@@ -38,16 +45,28 @@ In addition, configure scraping as follows:
 
 ### Installation
 
-Specify the Prometheus URL to `pvc-autoresizer` argument as `--prometheus-url`.
+You must configure how `pvc-autoresizer` obtains volume usage metrics. Two supported methods are described below.
+
+- Prometheus (HTTP endpoint): set the `--prometheus-url` argument to point to your Prometheus server endpoint.
+- Kubernetes Metrics API: enable use of the cluster Metrics API by setting the `--use-k8s-metrics-api=true` argument.
+
+`pvc-autoresizer` can be deployed to a Kubernetes cluster via `helm`.
+
+Example: install using Prometheus endpoint
+
+```sh
+helm repo add pvc-autoresizer https://topolvm.github.io/pvc-autoresizer/
+helm install --create-namespace --namespace pvc-autoresizer pvc-autoresizer pvc-autoresizer/pvc-autoresizer --set "controller.args.prometheusURL=<YOUR_PROMETHEUS_ENDPOINT>"
+```
 
-`pvc-autoresizer` can be deployed to a Kubernetes cluster via `helm`:
+Example: install using Kubernetes Metrics API
 
 ```sh
 helm repo add pvc-autoresizer https://topolvm.github.io/pvc-autoresizer/
-helm install --create-namespace --namespace pvc-autoresizer pvc-autoresizer pvc-autoresizer/pvc-autoresizer --set "controller.args.prometheusURL=<YOUR PROMETHEUS ENDPOINT>"
+helm install --create-namespace --namespace pvc-autoresizer pvc-autoresizer pvc-autoresizer/pvc-autoresizer --set "controller.args.useK8sMetricsApi=true"
 ```
 
-See the Chart [README.md](./charts/pvc-autoresizer/README.md) for detailed documentation on the Helm Chart.
+See the Chart [`charts/pvc-autoresizer/README.md`](./charts/pvc-autoresizer/README.md) for detailed Helm options and additional configuration.
```

Comment:
nits: It seems to me that "See the Chart charts/pvc-autoresizer/README.md" is redundant. Removing the first "Chart" or reverting to the former expression might be better.

Reply:
Since the original text also had overlapping "Chart", I removed the first one.
https://github.com/topolvm/pvc-autoresizer/compare/1062069951e827e8d97097afc4ca5e53289f2273..59a4e861e8ed8e00d296f89ff606217ebfe93524